SendGrid/SendGrid.IntegrationTests/MailServiceTest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SendGrid.Helpers.Mail;
+using System.Threading.Tasks;
+
+namespace SendGrid.IntegrationTests
+{
+    [TestClass]
+    public class MailServiceTest
+    {
+        private string _apiKey;
+        private SendGrid.SendGridAPIClient _client;
+
+        public MailServiceTest()
+        {
+            _apiKey = Environment.GetEnvironmentVariable("SENDGRID_APIKEY", EnvironmentVariableTarget.User);
+            _client = new SendGridAPIClient(_apiKey, "https://api.sendgrid.com");
+        }
+
+        [TestMethod]
+        public async Task SendMailToSingleRecipientTest_Success()
+        {
+            //Arrange
+            Email from = new Email("test@example.com");
+            String subject = "Hello World from the SendGrid CSharp Library";
+            Email to = new Email("test@example.com");
+            Content content = new Content("text/plain", "Textual content");
+            SendGrid.Models.Mail mail = new SendGrid.Models.Mail(from, subject, to, content);
+
+
+            //Act
+            _client.Mail.MailRequest = mail;
```

Comment:
How will this work in multi-threaded applications? If one thread writes to this variable whilst another thread is sending an email, you have a recipe for disaster. In it's current state it would be up to the consumer to implement locking on this variable. This should be handled internally and the library should be completely thread-safe.